src/content/contributing/translation-program/translation-versions/index.md

```diff
@@ -0,0 +1,131 @@
+- [Java](/developers/docs/programming-languages/java/)
+- [JavaScript](/developers/docs/programming-languages/javascript/)
+- [Python](/developers/docs/programming-languages/python/)
+- [Rust](/developers/docs/programming-languages/rust/)
+- [Standards](/developers/docs/standards/)
+- [Token standards](/developers/docs/standards/tokens/)
+- [ERC-20](/developers/docs/standards/tokens/erc-20/)
+- [ERC-721](/developers/docs/standards/tokens/erc-721/)
+- [Orcles](/developers/docs/oracles/)
+- [Scaling](/developers/docs/scaling/)
+- [Layer 2 Rollups](/developers/docs/scaling/layer-2-rollups/)
+- [State channels](/developers/docs/scaling/state-channels)
+- [Sidechains](/developers/docs/scaling/sidechains)
+- [Plasma](/developers/docs/scaling/plasma)
+- [Validium](/developers/docs/scaling/validium/)
+
+## Version 2.2 {#version-22}
+
+Language V2.2 translates a large collection of the [tutorials](/developers/tutorials) hosted on ethereum.org.
+
+### Tutorials {#tutorials}
+
+- [Using WebSockets](/developers/tutorials/using-websockets/)
+- [Waffle: Dynamic mocking and testing contract calls](/developers/tutorials/waffle-dynamic-mocking-and-testing-calls/)
+- [Getting Started with Ethereum Development](/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/)
+- [Waffle say hello world tutorial with hardhat and ethers](/developers/tutorials/waffle-hello-world-with-buidler-tutorial/)
+- [The Graph: Fixing Web3 data querying](/developers/tutorials/the-graph-fixing-web3-data-querying/)
+- [Smart contract security guidelines](/developers/tutorials/smart-contract-security-guidelines/)
+- [Token integration checklist](/developers/tutorials/token-integration-checklist/)
+- [How to run a light node with Geth](/developers/tutorials/run-light-node-geth/)
+- [How to use Slither to find smart contract bugs](/developers/tutorials/how-to-use-slither-to-find-smart-contract-bugs/)
+- [How to turn your Raspberry Pi 4 into a node just by flashing the MicroSD card](/developers/tutorials/run-node-raspberry-pi/)
+- [Calling a smart contract from JavaScript](/developers/tutorials/calling-a-smart-contract-from-javascript/)
+- [Transfers and approval of ERC-20 tokens from s solidity smart contract](/developers/tutorials/transfers-and-approval-of-erc-20-tokens-from-a-solidity-smart-contract/)
+- [Interact wit hother contracts from Solidity](/developers/tutorials/interact-with-other-contracts-from-solidity/)
+- [Understand the ERC-20 token smart contract](/developers/tutorials/understand-the-erc-20-token-smart-contract/)
+- [Deploying your first smart contract](/developers/tutorials/deploying-your-first-smart-contract/)
+- [Logging data from smart contracts with events](/developers/tutorials/logging-events-smart-contracts/)
+- [How to implement an ERC-721 market](/developers/tutorials/how-to-implement-an-erc721-market/)
+- [How to use Manticore to find bugs in smart contracts](/developers/tutorials/how-to-use-manticore-to-find-smart-contract-bugs/)
```

Comment:
Similar to above... should we consider just stating everything in "/developers/tutorials" is v2.2?

I suppose part of this depends on where newly listed tutorials and docs are going to be slotted for new content versions or not. If not, seems more dynamic and future proof to not explicitly list every page in these large folders.